src/neo/IO/Helper.cs

```diff
@@ -96,6 +142,11 @@ public static void FillBuffer(this BinaryReader reader, Span<byte> buffer)
             }
         }
 
+        /// <summary>
+        /// Gets the size of the size prefix of the data with the specified size.
```

Comment:
I'm not sure it's obvious what `size prefix` means here, to me that's just variable-length encoding of `value` integer. It's mostly used for array sizes, but it's not limited to that use only.

Reply:
Updated.